use v1 recipe and call make

danielnachun opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

danielnachun committed:
use v1 recipe and call make

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.11.15.40.30

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

danielnachun committed:
use rattler-build

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

danielnachun committed:
copy executabe

danielnachun committed:
fix copy on Windows

danielnachun committed:
more fixes

dhirschfeld commented:
Undefined symbols on `osx_64`:
```
 │ │ $BUILD_PREFIX/go/pkg/tool/darwin_amd64/link: running x86_64-apple-darwin13.4.0-clang failed: exit status 1
 │ │ $BUILD_PREFIX/bin/x86_64-apple-darwin13.4.0-clang -arch x86_64 -m64 -Wl,-S -Wl,-x -o $WORK/b001/exe/a.out -Qunused-arguments /var/folders/71/5myfy97j1pqgbz_z1v70yb2r0000gn/T/go-link-3569594004/go.o -lresolv -framework CoreFoundation -framework Security -Wl,-sectcreate,__TEXT,__info_plist,/tmp/micro-info.plist
 │ │ Undefined symbols for architecture x86_64:
 │ │   "_SecTrustCopyCertificateChain", referenced from:
 │ │       _crypto/x509/internal/macos.x509_SecTrustCopyCertificateChain_trampoline.abi0 in go.o
 │ │ ld: symbol(s) not found for architecture x86_64
 │ │ x86_64-apple-darwin13.4: error: linker command failed with exit code 1 (use -v to see invocation)
 ```

dhirschfeld commented:
In the same (`osx_64`) build it also seems to not be picking up the version:
```
 │ │ fatal: not a git repository (or any of the parent directories): .git
 │ │ 2025/09/11 18:16:07 semver.ParseTolerant(): strconv.ParseUint: parsing "": invalid syntax
 │ │ go build -trimpath -ldflags "-s -w -X github.com/zyedidia/micro/v2/internal/util.Version=0.0.0-unknown -X github.com/zyedidia/micro/v2/internal/util.CommitHash= -X 'github.com/zyedidia/micro/v2/internal/util.CompileDate=September 11, 2025' -linkmode external -extldflags -Wl,-sectcreate,__TEXT,__info_plist,/tmp/micro-info.plist" ./cmd/micro
 │ │ go: downloading github.com/go-errors/errors v1.0.1
 ```
 
 In the successful builds (e.g. `win_64`) those same flags are also not getting set correctly.

danielnachun committed:
try newer C stdlib

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.11.15.40.30

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

danielnachun committed:
try newer C stdlib

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.11.23.09.59

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

danielnachun committed:
try newer C stdlib

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.11.23.09.59

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

dhirschfeld commented:
An odd error:
```
++++ case "${MACOSX_SDK_VERSION}" in
++++ echo 'Unknown version & hash, please update conda-forge-ci-setup'\''s download_osx_sdk.sh'
+++ sdk_sha256='Unknown version & hash, please update conda-forge-ci-setup'\''s download_osx_sdk.sh'
+++ echo 'Unknown version & hash, please update conda-forge-ci-setup'\''s download_osx_sdk.sh *MacOSX12.7.sdk.tar.xz'
+++ shasum -a 256 -c
shasum: standard input: no properly formatted SHA checksum lines found
```

dhirschfeld commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17673364757](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17673364757). Examine the logs at this URL for more detail.</sub>

dhirschfeld commented:
Rerunning CI, just in case. Nothing else seems obviously wrong :/

danielnachun committed:
use a valid SDK version

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.12.13.06.32

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

danielnachun committed:
use a valid SDK version

danielnachun committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.12.13.06.32

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

danielnachun commented:
@dhirschfeld This is ready to merge.  The error from before was because after 11.0, macOS SDK versions must be specified using the minor version as well, and only certain minor versions are available.  So for macOS 12, the version must be 12.3.  I used the wrong one initially but it works now with the valid version.

dhirschfeld commented:
Thanks @danielnachun for getting to the bottom of it!
